blog/2024-05-07-weaviate-1-25-release/_core-1-25-include.mdx

```diff
@@ -4,8 +4,12 @@ Here are the release ⭐️*highlights*⭐️!
 
 ![Weaviate 1.25](./img/hero.png)
 
-- [**Dynamic Vector Index**](/blog/weaviate-1-25-release#dynamic-vectr-index) An index configuration that allows Weavaite to dyanamically switch from flat to HNSW as object count increases.
-- [**New Modules**](/blog/weaviate-1-25-release#new-modules) Loads of new modules that will allow you to use open source locally running and hosted embedding and language models!  
+- [**Dynamic Vector Index**](/blog/weaviate-1-25-release#dynamic-vector-index) Dynamically switch from flat indexes to HNSW to efficiently scale as your data grows.
+- [**Raft**](/blog/weaviate-1-25-release#raft) Improves schema management for faster updates and more reliable clusters. 
+- [**New Modules**](/blog/weaviate-1-25-release#new-modules) Loads of new modules that empower you to host and run open source embedding and language models locally!
+- [**Batch vectorization**](/blog/weaviate-1-25-release#batch-vectorization) Faster and more efficient use of APIs during data import.
+- [**Automatic tenant creation**](/blog/weaviate-1-25-release#automatic-tenant-creation) Easier data uploads and tenant creation for you application.
+- [**Search improvements**](/blog/weaviate-1-25-release#search-improvements) Hybrid search gets vector similarity search and grouping. method. Keyword search also gets grouping.
```

Comment:
It might be worth clarifying "vector similarity search", as vector search was already available. I think it's aspects like "move to/away", which allow more nuanced vector searches. 

How about "more nuanced" vector search, or "finer control over vector search"?